### PR TITLE
Create abstraction for lxqt desktop group

### PR DESCRIPTION
**Pull request by Besanon**
first file for the LXQT 2.0 desktop group

### CODE REVIEW

File: apparmor.d/abstractions/lxqt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  include <abstractions/wayland>
+  include <abstractions/X-strict>
+
+  signal (receive) set=(kill, term) peer=lxqt-session,	
+
+  /usr/share/hwdata/pnp.ids 	  r,
+  /usr/share/icu/@{int}.@{int}/*.dat 			r,
+  /usr/share/lxqt/**    				r,
+  /usr/share/qt{5,6}/ 					r,
+  /usr/share/qt{5,6}/{,**} 			r,
```
Comment (roddhjav) [inline apparmor.d/abstractions/lxqt:20]: Remove, they should already be included in `abstractions/qt5`.

File: apparmor.d/abstractions/lxqt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ 
+  owner @{HOME}/.Xdefaults 			r,
+
+  owner @{user_cache_dirs}/fontconfig/* 		      rw,
+  owner @{user_cache_dirs}/lxqt-notificationd/*  	r,
+
+  owner @{user_config_dirs}/lxqt/*.conf			      rw,
+
```
Comment (roddhjav) [inline apparmor.d/abstractions/lxqt:28]: I am not sure if allowing full config access to all program using this abs (ie: a lot) is the best option. I think we can start this way and restrict this a bit later.

**Comment by roddhjav** [comment]
Thanks, this will be easier to review/merge.

Tabs are not allowed, please replace then with spaces (this is why the CI failed).

**Comment by Besanon** [comment]
ok, i will leave that unchanged but i am going to test wether just allowing read will raise any issues

Am 22.10.2024 um 17:58, Alex ***@***.***> schrieb:

@roddhjav commented on this pull request.
In apparmor.d/abstractions/lxqt:> +  owner @{user_config_dirs}/lxqt/*.conf			      rw,+
I am not sure if allowing full config access to all program using this abs (ie: a lot) is the best option. I think we can start this way and restrict this a bit later.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

-- 
Sent with https://mailfence.com  
Secure and private email

**Comment by roddhjav** [comment]
Also, add: `include <abstractions/xdg-desktop>` it is a commonly used among other similar abstraction.

**Comment by Besanon** [comment]
owner @{user_config_dirs}/lxqt/*.conf                 rw,
should be left in the abstractions - most of the lxqt-files write to the conf files

**Comment by roddhjav** [comment]
Nice, thanks, merged!

**Comment by Besanon** [comment]
all right gonna send you startlxqt 

Am 22.10.2024 um 19:51, Alex ***@***.***> schrieb:

Nice, thanks, merged!
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

-- 
Sent with https://mailfence.com  
Secure and private email

**Pull request merged**